Blog post for unpacking case fatality rate

hlzhou opened the pull request:
I had trouble running blogdown and serving the post on my machine, but otherwise I followed the instructions on the README for where to place the blog post and dependencies.

capnrefsmmat commented on content/blog/2021-01-21-unpack-cfr.Rmd:
```diff
@@ -0,0 +1,249 @@
+
+Since the COVID-19 pandemic first reached the United States, the case fatality (CFR) rate has fallen considerably. Between the first peak in mid-April and the second peak in mid-July, the case fatality rate fell from $7.9\%$ to the $0.7\%$--$2.3\%$ range, where it has since remained despite cases rising again into an (ongoing) third wave:
+
+![cfr from covidcast](img/img_country/national_cases.svg)
+**Figure 1.** _From left to right: confirmed cases, deaths, and case fatality rate, calculated using 7-day trailing averages based on national reporting data available via USAFacts (@usafacts) and pulled from the CMU COVIDcast API (@covidcast2020api). Data outside the April 1st to December 1st time range considered in this study is grayed out._
```
I believe the blog style puts "cfr from covidcast" as the caption under the plot, so you can move the "Figure 1" text in there

hlzhou replied:
Got it, went ahead and made those changes.

capnrefsmmat commented on content/blog/2021-01-21-unpack-cfr.Rmd:
```diff
@@ -0,0 +1,249 @@
+
+Taking the above three adjustments into account, our primary quantity of interest for treatment improvements is the age-stratified HFR. For the rest of the post, 
+we define CFR and HFR at day $t$ as follows:
+\begin{align*}
+    \text{CFR}_t &= \frac{\text{cases confirmed (or reported) at day $t$ that eventually die}}{\text{cases confirmed (or reported) at day $t$}}\\
+    \text{HFR}_t &= \frac{\text{cases confirmed (or reported) at day $t$ that eventually get hospitalized and die}}{\text{cases confirmed (or reported) at day $t$ that eventually get hospitalized}}
+\end{align*}
```
We use KaTeX to render math in HTML, so this needs to be
```suggestion
$$
\begin{aligned}
    \text{CFR}_t &= \frac{\text{cases confirmed (or reported) at day $t$ that eventually die}}{\text{cases confirmed (or reported) at day $t$}}\\
    \text{HFR}_t &= \frac{\text{cases confirmed (or reported) at day $t$ that eventually get hospitalized and die}}{\text{cases confirmed (or reported) at day $t$ that eventually get hospitalized}}
\end{aligned}
$$
```

hlzhou replied:
Done.

capnrefsmmat commented on content/blog/2021-01-21-unpack-cfr.Rmd:
```diff
@@ -0,0 +1,249 @@
+Here, "eventually" means that the hospitalization or death was recorded by the date of data collection (i.e. December 31st), which gives each patient at least 30 days after their case date to have the events recorded.
+
+## Quantifying True Improvements
+Thus far, news and academic sources have highlighted three main ``true improvements": improvements in treatment (H4), disease mutations (H5), and reduced viral loads due to social distancing (H6). 
```
```suggestion
Thus far, news and academic sources have highlighted three main "true improvements": improvements in treatment (H4), disease mutations (H5), and reduced viral loads due to social distancing (H6). 
```

hlzhou replied:
Done

capnrefsmmat commented on content/blog/2021-01-21-unpack-cfr.Rmd:
```diff
@@ -0,0 +1,249 @@
+
+While we focus on the two peaks and the endpoints of the study time range, we also include plots of HFR estimates with uncertainty for all dates between April 1st and December 1st: 
+
+![hfr_florida](img/img1/florida_fdoh_full_est.svg)
+<p style="text-align: center;"><i>(a) Florida FDOH data</i></p>
+
+![hfr_florida](img/img_country/national_cleaned_full_est.svg)
+<p style="text-align: center;"><i>(b) United States CDC data</i></p>
```
these captions can again go into the square brackets, otherwise `hfr_florida` will appear underneath the images

hlzhou replied:
Done.

capnrefsmmat commented on content/blog/2021-01-21-unpack-cfr.Rmd:
```diff
@@ -0,0 +1,249 @@
+
+# Introduction
+
+Since the COVID-19 pandemic first reached the United States, the case fatality (CFR) rate has fallen considerably. Between the first peak in mid-April and the second peak in mid-July, the case fatality rate fell from $7.9\%$ to the $0.7\%$--$2.3\%$ range, where it has since remained despite cases rising again into an (ongoing) third wave:
```
the numbers here don't need to be in math mode

hlzhou replied:
Got it, fixed this and all similar cases.

capnrefsmmat reviewed (commented):
just a quick formatting check; I haven't read the text yet, but that can happen in the next few days

hlzhou commented on content/blog/2021-01-21-unpack-cfr.Rmd:
```diff
@@ -0,0 +1,249 @@
+author: "Helen Zhou, Cheng Cheng, Jeremy C. Weiss, Zachary C. Lipton"
+output:
+  bookdown::html_document2: default
+bibliography: refs.bib
```
Yes thanks for catching that, just added it to the latest commit.

capnrefsmmat commented:
I'll start trying to get this into our final formatting in the next few days. Could you provide a short one-sentence bio for each author, like the ones at [the end of the new causal post](https://delphi.cmu.edu/blog/2021/01/19/causal-inference-for-social-mobility-and-covid-19/)? Include a website if the author has one. You can post the bios in a comment here and I'll get them incorporated.

capnrefsmmat commented:
Also, a short 150-character summary to go in [the post list](https://delphi.cmu.edu/blog/).

You might also start thinking about a short tweet thread to accompany the post. You can see some examples @CmuDelphi; they usually highlight a couple key graphs and summarize the main results.

sgratzl commented on dependencies.R:
```diff
@@ -1,4 +1,5 @@
 # install.packages(c(), repo="http://cran.rstudio.com/")
+install.packages("bookdown", repo="http://cran.rstudio.com/")
```
shouldn't blogdown be already installed by the anaconda environment?

hlzhou replied:
Is bookdown included in blogdown? We added this line since we used bookdown for our citations.

sgratzl commented on content/blog/2021-01-21-unpack-cfr.Rmd:
```diff
@@ -0,0 +1,239 @@
+output:
+  bookdown::html_document2: default
+bibliography: unpack-cfr/refs.bib
+link-citations: yes
```
please add also the other header elements, see the other blog posts for example, including:

 * author list
 * acknowledgments
 * summary
 * hero image
 * tags
 * ...

see https://github.com/cmu-delphi/www-main/blob/8900c334d1b786ecf6ff04e2426241a727565f1d/content/blog/2021-01-15-causal-effect-mobility.Rmd#L1-L28

sgratzl commented on content/blog/2021-01-21-unpack-cfr.Rmd:
```diff
@@ -0,0 +1,239 @@
+
+Since the COVID-19 pandemic first reached the United States, the case fatality (CFR) rate has fallen considerably. Between the first peak in mid-April and the second peak in mid-July, the case fatality rate fell from 7.9% to the 0.7%--2.3% range, where it has since remained despite cases rising again into an (ongoing) third wave:
+
+![**Figure 1.** From left to right: confirmed cases, deaths, and case fatality rate, calculated using 7-day trailing averages based on national reporting data available via USAFacts (@usafacts) and pulled from the CMU COVIDcast API (@covidcast2020api). Data outside the April 1st to December 1st time range considered in this study is grayed out.](unpack-cfr/img_country/national_cases.svg)
```
can we move the figures to the same place as all the other figure and using the same notation? i.e. in /static/blog/.... instead of /content/...

hlzhou commented:
> I'll start trying to get this into our final formatting in the next few days. Could you provide a short one-sentence bio for each author, like the ones at [the end of the new causal post](https://delphi.cmu.edu/blog/2021/01/19/causal-inference-for-social-mobility-and-covid-19/)? Include a website if the author has one. You can post the bios in a comment here and I'll get them incorporated.

Sure thing, here are the bios:
(Helen Zhou)[https://www.helen-zhou.com] is a PhD student in the Machine Learning Department at CMU.

(Cheng Cheng)[https://linkedin.com/in/cheng-cheng-419718106] is a PhD student in Machine Learning and Public Policy at CMU.

(Jeremy Chen Weiss)[https://www.andrew.cmu.edu/user/jweiss2/] is an Assistant Professor of Health Informatics at CMU and is a member of Delphi.

(Zachary Chase Lipton)[acmilab.org] is an Assistant Professor of Operations Research and Machine Learning at CMU and is a member of Delphi.

hlzhou commented:
> Also, a short 150-character summary to go in [the post list](https://delphi.cmu.edu/blog/).
> 
> You might also start thinking about a short tweet thread to accompany the post. You can see some examples @CmuDelphi; they usually highlight a couple key graphs and summarize the main results.

Here's a 149-character summary:
Since the pandemic started, case fatality has plummeted. How can we unpack treatment improvements versus younger patients, increased testing, etc.?

sgratzl commented:
I just updated the setup of this blog post and compiled a proper version. 

changed
 * updated to latest dev
 * added author infos
 * added summary
 * fixed the blogdown setup - somehow the bib file couldn't be parsed

TODO
 * hero image?

sgratzl reviewed (commented):
setup should be fine now

tildechris commented:
Is there a setting to give figures a white background instead of transparent?  Here's what it looks like when they are enlarged:

![image](https://user-images.githubusercontent.com/46794497/105507706-85d4ce80-5c99-11eb-8a06-2417025ea912.png)

sgratzl reviewed (commented):
setup should be fine now

sgratzl commented:
> Is there a setting to give figures a white background instead of transparent? Here's what it looks like when they are enlarged:

we can set a background directly in the SVG files. If we would do it in the HTML popup, the white area would cover the whole screen.

sgratzl commented:
![image](https://user-images.githubusercontent.com/4129778/105508513-b6822b80-5ccc-11eb-8bef-44b986a7cb3f.png)

should we keep enforcing that this text is in blue?

capnrefsmmat commented:
I'm not a fan of using colored text here; the post already uses bold, italics, and bold-italic text, and four emphasis styles is a bit too much.

Also, I'd change the top-level headings to `##`, since we usually use `<h2>`s for headers in content (since the post title is an `<h1>`). Subheads should go down a level as well.

sgratzl commented:
> I'm not a fan of using colored text here; the post already uses bold, italics, and bold-italic text, and four emphasis styles is a bit too much.
> 
> Also, I'd change the top-level headings to `##`, since we usually use `<h2>`s for headers in content (since the post title is an `<h1>`). Subheads should go down a level as well.

done

hlzhou commented:
> I just updated the setup of this blog post and compiled a proper version.
> 
> changed
> 
> * updated to latest dev
> * added author infos
> * added summary
> * fixed the blogdown setup - somehow the bib file couldn't be parsed
> 
> TODO
> 
> * hero image?
> * tags?
> * should we try to style the bibliography a bit?
> 
> current version: https://deploy-preview-226--cmu-delphi-main.netlify.app/blog/2021/01/22/unpacking-the-drop-in-covid-19-case-fatality-rates/

I added an APA bibliography style for now, some potential new tags (not sure if it falls under any of the previous tags), and an acknowledgements section. Where do you normally find hero images?

tildechris commented:
> Where do you normally find hero images?

Two best royalty free sites are unsplash.com and pexels.com.  You can also message me on slack and I can help if you have a rough idea of what you're looking for.

tildechris commented:
What do you think about this image?  I wanted to evoke a "one out X" sentiment.

![one-out-of](https://user-images.githubusercontent.com/46794497/105553518-8c803780-5cd3-11eb-91fd-9d1af38ad8a8.jpg)

capnrefsmmat commented:
Dunno, when the topic is case fatality rate, isn't singling out one person in the image a bit morbid?

hlzhou commented:
Thanks for the pointers! How do you feel about this photo?

hlzhou commented:
Hmm how do you feel about this photo? I screenshotted it from a Department of Defense video of a press briefing, and edited it.
![image](https://user-images.githubusercontent.com/6379568/105558105-d3722b00-5cdb-11eb-9a06-8f0c40240f4d.png)

Not sure if it's high quality enough though, so here are some other options
https://unsplash.com/photos/ewtkO_zvYXw
https://www.pexels.com/photo/blue-and-silver-stetoscope-40568/
https://www.pexels.com/photo/mother-putting-a-face-mask-on-her-daughter-4261252/
https://unsplash.com/photos/zbpgmGe27p8
https://unsplash.com/photos/yGUuMIqjIrU
https://unsplash.com/photos/Ei46GUcG_y4

hlzhou commented:
Just added a hero image under the static/blog/images folder titled "unpack-cfr-full-size.jpg" and "unpack-cfr-thumb.jpg." Are there any other remaining TODOs for this blog post?

sgratzl commented:
current state: https://deploy-preview-226--cmu-delphi-main.netlify.app/blog/2021/01/22/unpacking-the-drop-in-covid-19-case-fatality-rates/

sgratzl reviewed (approved):
ok from a technical perspective

tildechris commented:
> Dunno, when the topic is case fatality rate, isn't singling out one person in the image a bit morbid?

Well, the topic is literally about mortality, but yes perhaps it's too evocative.  I saw the hospital room hero image, and I like that better.  LGTM from me too, so we just need to finalize the content.

zackchase commented:
hi all, after making a pass (see my commit) I approve the content and am happy for the post to be deployed provided that engineering confirms that everything renders properly and that my line breaking didn't screw up anything.

zackchase reviewed (approved):
Provided that this passes engineering checks for rendering properly to HTML, I approve this content.

tildechris reviewed (approved):
LGTM.
